library/ovirt_vm.py

```diff
@@ -10,7 +10,7 @@
 
```

Comment:
ovirt_vms.py should be completely removed from the project, so the role will use ovirt_vm module included in Ansible itself.

Reply:
it will be removed in a different PR, this one only handles the renaming

Reply:
Why to bother with renaming, when you are going to remove the file completely in next PR?

Reply:
Yes that as well. All should be renamed.

Reply:
> It requires to verify that all the needed functionality exists on the ovirt_vm module in ansible and changing to code to use it - we currently don't have the capacity for it.

This needs to be delivered in 4.2.8, because this is the last 4.2.z version and we will not have any way how to fix 4.2.z after 4.3 release. If this is not included 4.2.8, that the role will be broken when Ansible 2.8 is released in May.

According to README the role requires ovirt_vm module from Ansible 2.6.1, so requiring Ansible 2.7.2 means that you have all your requirements included (Ansible modules are backward compatible).

Reply:
removed